metadata-editor/app/lib/DynamoEdits.scala

```diff
@@ -0,0 +1,27 @@
+package lib
+
+import com.gu.mediaservice.lib.aws.DynamoDB
+import com.gu.mediaservice.model.Edits
+import play.api.libs.json.{Json, JsObject}
+import play.api.libs.concurrent.Execution.Implicits._
+
+object DynamoEdits {
```

Comment:
Can we have a comment about what this is and why it's here (vs the controller), as it looks transitional (for a reason)?